recipes/libtorch/1.2.0/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conans import ConanFile, CMake, tools
+
+class Libtorch(ConanFile):
+    name = "libtorch"
+    version = "1.2.0"
+    license = "BSD"
+    url = "https://pytorch.org/"
+    description = "PYTORCH C++ API"
+    settings = {"os": ["Windows","Linux"],
+        "build_type": ["Debug", "Release", "RelWithDebInfo"],
+        "compiler": {"gcc": {"version": None,
+                            "libcxx": ["libstdc++", "libstdc++11"]},
+                    "Visual Studio": {"version": None},
+                    }
+        }
+
+    options = {"cuda": ["9.2","10.0", "None"]}
+    default_options = {"cuda": "None"}
+
+    generators = "cmake"
+    url_base = "https://download.pytorch.org/libtorch/"
+
+
+    def getGPU(self):
+        if(self.options.cuda == "9.2"):
+            return "cu92/"
+        if(self.options.cuda == "10.0"):
+            return "cu100/"
+        else:
+            return "cpu/"
+
+    def getOS(self):
+        if(self.settings.os == "Windows"):
+            if(self.settings.build_type == "Debug"):
+                return "libtorch-win-shared-with-deps-debug-"
+            return "libtorch-win-shared-with-deps-"
+        if(self.settings.os == "Linux"):
+            if(self.settings.compiler.libcxx == "libstdc++11"):
+                return "libtorch-cxx11-abi-shared-with-deps-"
+            return "libtorch-shared-with-deps-"
+    
+    def build(self):
```

Comment:
There shouldn't be downloads in `build()` please do this in `source()`

Reply:
renamed the build(self) into source(self)